doc/deployment/deploy-to-azure-web-app-javase-using-jenkins.md

```diff
@@ -10,16 +10,18 @@ Below are the major steps in this tutorial.
 
 ## <a name="create-app"></a>Create Azure Web App
 
-Create an Azure Web App by [Azure portal](https://github.com/Azure/azure-docs-sdk-java/blob/master/docs-ref-conceptual/spring-framework/deploy-spring-boot-java-web-app-on-azure.md#create-an-azure-web-app-for-use-with-java) or [Azure CLI](https://docs.microsoft.com/en-us/azure/app-service/app-service-cli-samples?toc=%2fazure%2fapp-service%2fcontainers%2ftoc.json)
+Create an Azure Web App with Java as a rumtime stack by [Azure portal](https://azure.microsoft.com/en-us/blog/public-preview-of-java-on-app-service-built-in-support-for-tomcat-and-openjdk) or [Azure CLI](https://docs.microsoft.com/en-us/azure/app-service/app-service-cli-samples?toc=%2fazure%2fapp-service%2fcontainers%2ftoc.json)
 
 > If you use Web App on Linux with jre runtime stack, you have to build your app with port 80 or set the web app setting with `PORT=8080`
 
 ## <a name="prepare"></a>Prepare Jenkins server
 
 1. Deploy a [Jenkins Master](https://aka.ms/jenkins-on-azure) on Azure
 
+1. Install maven and zip commands for your Jenkins Master.
```

Comment:
Can we give instructions or links here?

Reply:
They are common commands, just a reminder here, won't be a challenge for users to install them.